framework/components/ecl-forms/ecl-forms-legends/_legends.scss

```diff
@@ -1,13 +1,10 @@
-/*
-Legend
+/**
+* Legend component
+* @define form-legend
 */
 
 $form-item-legend-font-size: map-get($ecl-font-size, 'xs');
 
-.legend {
+.ecl-form-legend {
```

Comment:
Ex: we have 2 labels components (ecl-label and ecl-form-label) this legend is specific for form element so it seems more future proof and descriptive to stick to ecl-form-legend.